setup.cfg

```diff
@@ -23,7 +23,7 @@ classifier =
     Topic :: Software Development
 
 [options]
-python_requires = >= 3.7
+python_requires = >= 3.7, <3.11
```

Comment:
<3.11 due to a numba dependency requirement

Reply:
Hi @Scott-Simmons - does this still hold? And even if yes, tsfresh itself does not have any constraints, but its dependencies (numba in this case) has. I would therefore not like to constrain tsfresh itself, because the constraint should be transitive.

Reply:
Does not hold anymore. Have updated (ended up rebasing)